src/allocator.f90

```diff
@@ -1,37 +1,37 @@
-module m_allocator
```

Comment:
This is also interesting, even the module declaration starts with two spacing. Is there a way to disable it? seems like a waste of two spaces to me.